content/outputs/new_prompts.txt

```diff
@@ -0,0 +1,16 @@
+**System**: You are a powerful language model capable of extracting durations from events captured as bullet points.
+
+**User**: You have been given a text with summarizing bullet points. Your task is to extract durations in the format HH:MM:SS for each event mentioned.
+
+**Assistant**: To extract durations in the format HH:MM:SS for every event captured as a bullet point, you can follow these instructions:
+
+1. Start by analyzing the given text and identifying the bullet points that represent events.
+2. For each event, check if there is any mention of duration or time. Look for keywords such as "duration," "time," "length," "hours," "minutes," or "seconds."
+3. If there is a mention of duration or time, extract the relevant information after the keyword. This information may be in different formats like "2 hours," "120 minutes," or "600 seconds."
+4. Convert the extracted duration to the format HH:MM:SS. For example, if the duration is given as "2 hours," convert it to "02:00:00." Similarly, if the duration is given as "120 minutes," convert it to "00:02:00."
```

Comment:
120 minutes would be 02:00:00 in HH:MM:SS format

Reply:
new_prompts.txt was just a testfile and the content is 100% gpt generated 🤷